app/app.go

```diff
@@ -598,17 +601,17 @@ func New(
 	// If evidence needs to be handled for the app, set routes in router here and seal
 	app.EvidenceKeeper = *evidenceKeeper
 
-	// app.SchedulerKeeper = *schedulermodulekeeper.NewKeeper(
-	// 	appCodec,
-	// 	keys[schedulermoduletypes.StoreKey],
-	// 	memKeys[schedulermoduletypes.MemStoreKey],
-	// 	app.GetSubspace(schedulermoduletypes.ModuleName),
-	// 	app.AccountKeeper,
-	// 	app.EvmKeeper,
-	// 	[]xchain.Bridge{
-	// 		app.EvmKeeper,
-	// 	},
-	// )
+	app.SchedulerKeeper = *schedulermodulekeeper.NewKeeper(
+		appCodec,
+		runtime.NewKVStoreService(keys[evidencetypes.StoreKey]),
+		memKeys[schedulermoduletypes.MemStoreKey],
+		app.AccountKeeper,
+		nil,
```

Comment:
EVM module is enabled. You should pass EVM module keeper here.

Reply:
enabled and pushed